packages/serverpod/lib/src/authentication/authentication_info.dart

```diff
@@ -21,5 +29,10 @@ class AuthenticationInfo {
   final String? authId;
 
   /// Creates a new [AuthenticationInfo].
-  AuthenticationInfo(this.userId, this.scopes, {this.authId});
+  AuthenticationInfo(
+    @Deprecated('User parameter `user` instead') int? userId,
+    this.scopes, {
+    this.authId,
+    String? user,
+  }) : user = user ?? userId!.toString();
```

Comment:
Should we guard against both `userId` and `user` being passed?

This just checks that at least one is given.